src/components/common/SendCode/SendCode.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <component
+      :is="codeLayout"
+      :is-loading="isLoading"
+      :is-locked.sync="isLocked"
+      :counter="counter"
+      v-on="$listeners"
```

Comment:
try not to use v-on="$listeners" in future and declare methods for more understand of component api usage.
except some components or atoms in `ui-kit`
just define here `@send-code=onSend` and set `isLocked = true` in `onSend` method like
```
onSend() {
   this.isLocked = true;
}
```